### ISSUE TITLE
[Windows Terminal]: Narrator only announce 'space space' when user navigate through commands using up or down arrow keys.

### ISSUE THREAD

**Issue by ghost** [microsoft/terminal#11911]
### Windows Terminal version

1.12.2931.0

### Windows build number

10.0.22504.1010

### Other Software

**Test Environment:**
OS: Windows 11 Version Dev (OS Build 22504.1010)
App: Windows Terminal Preview
Screen Reader: Narrator
Tool: Accessibility Insights for Windows Version 1.1.1741.1

### Steps to reproduce

**Repro Steps:**
1. Open Windows Terminal.
3. Open Narrator using 'Win + Ctrl + Enter' key and ensure Scan mode is OFF.(Turn on Scan on mode using 'Caps lock + Space' keys).
4. Inside Command Prompt/PowerShell, Execute different commands then navigate through used commands using 'up' or 'down arrow' keys.
5. Observe the issue.

**User Experience:**
Screen Reader user will be impacted here if they didn't get the information which appear on screen and announcing incorrect information.

**Guideline Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

**Attachments:**
[Narrator.only.announce.space.space.....zip](https://github.com/microsoft/terminal/files/7690181/Narrator.only.announce.space.space.zip)

### Expected Behavior

Narrator should announce command name when user navigate through commands using up or down arrow keys.

### Actual Behavior

Narrator only announce 'space space' when user navigate through commands using up or down arrow keys. When we navigate using up arrow key then issue only repro when previous and next command initials are same.

**Note Issue:** Same issue repro when we press tab or Shift+ tab to navigate through all directory names.

**Observation:** Issue not repro with NVDA Screen Reader.

**Comment by carlos-zamora** [comment]
Tested this today on version 1.21.2361.0 and it no longer repros. We must've fixed this a while back.

This also should have probably been classified as higher severity. Looking at the video, it's pretty bad that we weren't reading changes to the prompt. Good thing it's fixed!
